### PR TITLE
docs: update user/README.md

### PR DESCRIPTION
**Pull request by nrc**
Update the output of TiUp and add links to TiDB docs

Tests 

 - No code

Release notes:

```release-note
NONE
```

### CODE REVIEW

File: doc/user/README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 For detailed documentation, see the manual which starts with an [overview](overview.md).
 
+To get started with TiDB using TiUp, see the [TiDB quick start guide](https://docs.pingcap.com/tidb/stable/quick-start-with-tidb). To deploy TiDB using TiUp in a production-like environment, see the [TiDB deploymeny guide](https://docs.pingcap.com/tidb/stable/production-deployment-using-tiup).
```
Comment (lucklove) [inline doc/user/README.md:7]:
```suggestion
To get started with TiDB using TiUP, see the [TiDB quick start guide](https://docs.pingcap.com/tidb/stable/quick-start-with-tidb). To deploy TiDB using TiUP in a production-like environment, see the [TiDB deploymeny guide](https://docs.pingcap.com/tidb/stable/production-deployment-using-tiup).
```

I know maybe `TiUp` is better than `TiUP` but we'd better keep the document consistent.

**Review by lucklove — APPROVED** [review]
LGTM

**Comment by lonng** [comment]
@nrc Please sign the CLA

**Pull request merged**